nih-sparc/sparc-app
# Underline page numbers #6ygpvn

# Description

Current page is now underlined in pager site-wide for better visibility.

## Tickets
[Clickup](https://app.clickup.com/t/6ygpvn)
[Wrike](https://www.wrike.com/workspace.htm?acc=3203588#path=folder&t=467384171&a=3203588&id=473707106&st=space-441356195)

## Type of change


# How Has This Been Tested?

1. Navigate to the Find Data page.
2. Current page should be underlined in pager.
3. Click on a different page.
4. Current page should be underlined in pager.

Same test applies for Resources tab.


# Checklist:
